format/base.go

```diff
@@ -151,37 +151,37 @@ func escape(text string) string {
 
 	// Escape the end of the string after the last URL if there's anything left
 	if len(b) > cursor {
-		builder.Write(escapeRaw(b[cursor:]))
+		builder.Write(EscapeRaw(b[cursor:]))
 	}
 
 	return builder.String()
 }
 
-func escapeRaw(segment []byte) []byte {
+func EscapeRaw(segment []byte) []byte {
```

Comment:
I don't think this needs to be exported. I'd rather leave it unexported and then export it later if needed